### PR TITLE
Fixed create_operatorhub_pr script along with other minor change in E2E test script

### PR DESCRIPTION
**Pull request by rajusem**
PR contains following changes 
- Fixed `hack/create_operatorhub_pr.sh` script for creating valid operator, bundle, catalog images 
- Created `AnomalyEngine` sample file into `/tests/sample/` folder used for E2E test. 
- Changes in E2E test script - `tests/run-e2e.sh`
  - Removed extra logs added to debug Prow CI job 
  - Using `AnomalyEngine` sample file from `/tests/sample/` folder.
  - Increased configmap count from `600` to `800`
- Corrected `make run` command issue
- Updated `README.md` file

Signed-off-by: Raj Zalavadia [rzalavad@redhat.com](mailto:rzalavad@redhat.com)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,6 +112,28 @@ UnDeploy the controller from the cluster:
 make undeploy
 ```
 
+### How to test oeprator bundle  in the cluster
```
Comment (Anxhela21) [inline README.md]:
```suggestion
### How to test operator bundle  in the cluster
```

File: README.md [hunk 2]
```diff
@@ -112,6 +112,28 @@ UnDeploy the controller from the cluster:
...
+# run bundle -> creates an index image referencing your bundle image, create a catalog source, 
+# deploy your Operator to your cluster by creating an OperatorGroup, Subscription, InstallPlan, 
+# and all other required resources
+operator-sdk run bundle quay.io/openshiftanalytics/observability-analytics-operator-bundle:0.1.0
+
+# cleanup installed operator -> cleanup OperatorGroup, Subscription, InstallPlan, Catalog, etc
+operator-sdk cleanup analytics-operator
+```
+
+### How to test oeprator catalog  in the cluster
```
Comment (Anxhela21) [inline README.md]:
```suggestion
### How to test operator catalog  in the cluster
```

File: README.md (hunk 1 above)
Comment (rajusem) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (rajusem) [reply README.md]: done

**Review by Anxhela21 — APPROVED** [review]
some minor comments otherwise looks good. Please feel free to remove hold when ready to merge.
/lgtm
/hold

**Comment by rajusem** [comment]
> some minor comments otherwise looks good. Please feel free to remove hold when ready to merge.

Corrected Typo.

**Comment by rajusem** [comment]
\unhold

**Pull request merged**